pkgs/native_assets_cli/example/link/package_with_assets/hook/link.dart

```diff
@@ -6,11 +6,7 @@ import 'package:native_assets_cli/native_assets_cli.dart';
 
 void main(List<String> args) async {
   await link(args, (config, output) async {
-    final assetsWithResource = config.assets.whereType<DataAsset>().where(
-        (asset) =>
-            config.treeshakingInformation
-                ?.any((resource) => resource.metadata == asset.name) ??
-            true);
-    output.addAssets(assetsWithResource);
+    final dataAssets = config.assets.whereType<DataAsset>();
```

Comment:
Maybe leave a TODO to add it back in.
This is an example, so people might read it in between landing this PR and the support for the tree shaking, which would confuse users.